tools/build_defs/configure_script.bzl

```diff
@@ -52,8 +53,8 @@ def _define_deps_flags(inputs):
         include_dirs[include_dir] = 1
 
     return {
-        "LDFLAGS": ["-L" + dir for dir in lib_dirs],
-        "CPPFLAGS": ["-I" + dir for dir in include_dirs],
+        "LDFLAGS": ["-L$EXT_BUILD_ROOT/" + dir for dir in lib_dirs],
```

Comment:
I may have missed something, what caused the need for this change?

Reply:
There was a bug, the paths were wrong - in Starlark, we always get the paths relative to some roots. So for the external code/script, I need to pass the full path.